version.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE version SYSTEM "../../../lib/pkp/dtd/pluginVersion.dtd">
```

Comment:
This path does not exist in this repository - I assume that is not a problem? Or exchange the dtd URL with a correct online URL?

Reply:
This is a default of the ojs repository. In the OJS repository the path leads to the plugin version. So I think its correct.